### PR TITLE
[#156610545] feat: added notification hub

### PR DESCRIPTION
**Pull request by gunzip**
we need gcm and apns credentials

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * ⚙️ [#156610545](https://www.pivotaltracker.com/story/show/156610545): Va creata risorsa Notification Hub su Azure

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: infrastructure/env/test/tfvars.json [hunk 1]
```diff
@@ -7,5 +7,6 @@
   "app_service_portal_post_logout_url": "https://agid-apim-test.portal.azure-api.net/",
   "azurerm_apim_sku": "Standard",
   "azurerm_kubernetes_agent_count": 3,
-  "azurerm_kubernetes_agent_vm_size": "Standard_D2"
+  "azurerm_kubernetes_agent_vm_size": "Standard_D2",
+  "azurerm_notification_hub_sku": "Free"
```
Comment (cloudify) [inline infrastructure/env/test/tfvars.json]: does the free version have all we need to develop/test the full functionality?

File: infrastructure/local-provisioners/azurerm_notification_hub.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+/**
+ * ApiParams contains the variables taken from command line (provisioner arguments)
+ * that represents values that *change* between different deploying environments.
+ */
+const ApimParams = t.interface({
+  location: t.string,
+  azurerm_resource_group: t.string,
+  azurerm_notification_hub: t.string,
+  azurerm_notification_hub_ns: t.string,
+  azurerm_notification_hub_sku: t.string
```
Comment (cloudify) [inline infrastructure/local-provisioners/azurerm_notification_hub.ts]: perhaps use an enum here?

Comment (gunzip) [reply infrastructure/local-provisioners/azurerm_notification_hub.ts]: done

**Pull request merged**